src/styles/base/_headers.scss

```diff
@@ -0,0 +1,3 @@
+h1, h2, h3, h4, h5 {
```

Comment:
I understand that this is prepared for later upgrade ?

Reply:
yup, there will be some global styles for each header. Mainly related to typography.